### DISCUSSION TITLE
Module "process" has been externalized for browser compatibility and cannot be accessed in client code.

### DISCUSSION THREAD

**Discussion by mirkofisic** [quasarframework/quasar#14379] [CLI - SSR mode] [answered]
I don't know if this is a bug, please anyone can help with this problem:

![image](https://user-images.githubusercontent.com/6698925/189892979-a6ffb3f2-5e84-48e1-a586-81e09d5df33c.png)

I used SSR mode but this happens when code is execute on client side:
![image](https://user-images.githubusercontent.com/6698925/189893468-4359cf90-d34d-4318-96fa-96481d20b947.png)

Quasar 2.7.7
NodeJS 16.14.0

**Comment by MilosPaunovic** [comment, marked as answer]
You don't need to import `process`.

**Reply by mirkofisic** [reply]
Oh sorry you are right, I did the import, but I don't know how, thank you very much
